src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1940,4 +1940,8 @@ Additional information: {1}</value>
   <data name="Prefer_top_level_statements" xml:space="preserve">
     <value>Prefer top-level statements</value>
   </data>
+  <data name="_0_1" xml:space="preserve">
```

Comment:
How about using the image moniker instead of the string to indicate the language name?
![image](https://user-images.githubusercontent.com/24360909/165637175-8a53a7f9-adbd-4fd5-9ae1-19c244786035.png)

Reply:
i'm not opposed to that.  if you want to teach me how to do that :)

Reply:
Working on that, I will ping you once I get a draft

Reply:
![image](https://user-images.githubusercontent.com/24360909/165647101-623339ac-4c1c-405c-97cb-789971fb2d94.png)

Get it : )